lib/Utils/Systemctl.pm

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package Utils::Systemctl;
+
+use base Exporter;
+use Exporter;
+
+use strict;
+use warnings;
+
+use utils 'systemctl';
```

Comment:
I'd move systemctl to this package also.

Reply:
Yes, but when I was editing the file number 186 (and this without verification runs and without checking if the general `use utils;` was still necessary after adding use `Utils::Systemd;'...) and I saw that the scroll bar wasn't even in the middle, I thought of creating another ticket for that. We have to save the world another day. I have kept the changes on my git stash.